### PR TITLE
DataFrame.to_string

### PR DESCRIPTION
**Pull request by rxin**
Part of #169

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -258,6 +258,53 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True, index=Tru
         return validate_arguments_and_invoke_function(
             kdf.to_pandas(), self.to_html, pd.DataFrame.to_html, args)
 
+    def to_string(self, buf=None, na_rep='NaN', float_format=None, header=True,
+                  index=True, length=False, dtype=False, name=False,
+                  max_rows=None):
+        """
+        Render a string representation of the Series.
+
+        .. note:: This method should only be used if the resulting Pandas DataFrame is expected
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: Actually, we're mixing NumPy docstring `Paramters` with underscores, and typical sphinx docstring (like `.. note::`) ..

If I am not mistaken, we can do as:

```

        Notes
        -----
        This method should only be used if the resulting Pandas DataFrame is expected
```

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -258,6 +258,53 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True, index=Tru
...
+        name : boolean, default False
+            Add the Series name if not None
+        max_rows : int, optional
+            Maximum number of rows to show before truncating. If None, show
+            all.
+
+        Returns
+        -------
+        formatted : string (if not buffer passed)
+        """
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: @rxin, can we add a doctest (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_string.html#pandas.DataFrame.to_string)?

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -258,6 +258,53 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True, index=Tru
...
+            Add the Series dtype
+        name : boolean, default False
+            Add the Series name if not None
+        max_rows : int, optional
+            Maximum number of rows to show before truncating. If None, show
+            all.
+
+        Returns
+        -------
+        formatted : string (if not buffer passed)
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: Seems this isn't matched with Pandas's doc (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_string.html#pandas.DataFrame.to_string)

**Comment by HyukjinKwon** [comment]
Looks good otherwise.

File: databricks/koalas/frame.py (hunk 1 above)
Comment (rxin) [reply databricks/koalas/frame.py]: Ah ok. They both work. We have other places too. I'm thinking we should just submit a separate PR to address all of them at the same time. Sounds good?

Comment (rxin) [reply databricks/koalas/frame.py]: actually i copied the series one haha. going to fix it.

**Comment by rxin** [comment]
@HyukjinKwon I pushed a new version. I'm actually thinking the other notes method is better, because it highlights the problem more.

See below:

Screen Shot 2019-04-30 at 8 12 04 PM

vs

Screen Shot 2019-04-30 at 8 12 09 PM

**Comment by HyukjinKwon** [comment]
Yup, either way is fine to me.

**Pull request merged**